web/modules/custom/mymodule/mymodule.module

```diff
@@ -0,0 +1,11 @@
+<?php
+
+/**
+ * @file
+ * Module file of custom module.
+ */
+
+function myplugin_preprocess_page(&$variables) {
+  $variables['#attached']['library'][] = 'myplugin/myplugin';
```

Comment:
@anitha-innoraft - This library should be attached in the formatter itself otherwise it will be included on all pages. We should always try to add libraries within the scope of their package/element.

Reply:
Done. Added inside the formatter.